src/core/crypto_tls.c

```diff
@@ -995,18 +995,21 @@ QuicCryptoTlsDecodeTransportParameters(
     _In_reads_(TPLen)
         const uint8_t* TPBuf,
     _In_ uint16_t TPLen,
+    _In_ BOOLEAN ClearExisting,
```

Comment:
I'm not sure this makes sense. If you're decoding new transport parameters, you should always overwrite.